Data/HashSet/Base.hs

```diff
@@ -259,38 +259,63 @@ fromListConstr = mkConstr hashSetDataType "fromList" [] Prefix
 hashSetDataType :: DataType
 hashSetDataType = mkDataType "Data.HashSet.Base.HashSet" [fromListConstr]
 
--- | /O(1)/ Construct an empty set.
+-- | Construct an empty set.
+--
+-- >>> HashSet.empty
```

Comment:
I think the qualification is unnecessary for these examples.

```suggestion
-- >>> empty
```

Reply:
I left it qualified because the container types should always be imported qualified because their names conflict, and `empty` also conflicts with the function from Control.Applicative. It would also be strange to use qualified functions everywhere else except for here.

Reply:
Fair enough.